source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -131,7 +131,52 @@ def get_backend_class(self, obj):
 
 
 class RidesharingServicesSerializer(OutsideServiceCommon):
-    pass
+    rating_scale_min = MethodField(schema_type=int, display_none=False)
+    rating_scale_max = MethodField(schema_type=int, display_none=False)
+    crowfly_radius = MethodField(schema_type=int, display_none=False)
+    network = MethodField(schema_type=str, display_none=False)
+
+    def get_rating_scale_min(self, obj):
+        return obj.get('rating_scale_min', None)
+
+    def get_rating_scale_max(self, obj):
+        return obj.get('rating_scale_max', None)
+
+    def get_crowfly_radius(self, obj):
+        return obj.get('crowfly_radius', None)
+
+    def get_network(self, obj):
+        return obj.get('network', None)
+
+
+class EquipmentProvidersSerializer(NullableDictSerializer):
+    key = MethodField(schema_type=str, display_none=False)
+    codes_types = MethodField(schema_type=str, many=True, display_none=True)
+    timeout = MethodField(schema_type=int, display_none=False)
+    fail_max = MethodField(schema_type=int, display_none=False)
+
+    def get_key(self, obj):
+        return obj.get('key', None)
```

Comment:
if we use `Field`, it is not allowed to overload the parameter. Overload works with MethodField because it inheritances of `serpy.MethodFiled`. Only `serpy.MethodFiled` allows to overload function for parameters.
Thererby, if we want optional field, you have to use it. 

The question is more,` what parameters we want optional ?`
After a ride inside the code, i thinks it is possible to use some parameters in mandatory mode. I'll try.